guides/common/modules/proc_installing-the-kernelcare-plug-in.adoc

```diff
@@ -0,0 +1,12 @@
+[id="Installing_the_KernelCare_Plug-in_{context}"]
+= Installing the KernelCare Plug-in
+
+Use the following procedure to install the KernelCare plug-in.
+
+.Procedure
+* Install the plug-in on your {ProjectServer}:
```

Comment:
I think the format should be followed even in one step procedures.
```suggestion
. Install the plug-in on your {ProjectServer}:
```

Reply:
Not applied, because this is the only bullet point.